vignettes/renv.Rmd

```diff
@@ -19,95 +15,125 @@ knitr::opts_chunk$set(
 )
 ```
 
-The `renv` package is a new effort to bring project-local R dependency
-management to your projects. The goal is for `renv` to be a robust, stable
-replacement for the [Packrat](https://rstudio.github.io/packrat/) package, with
-fewer surprises and better default behaviors.
+The `renv` package helps you create create **r**eproducible **env**ironments for your R projects.
+This vignette introduces you to the basic workflow of `renv`, showing you how to use the most important functions: `renv::init()`, `renv::snapshot()`, and `renv::restore()`.
+You'll also learn about some of the infrastructure that makes `renv` tick, some problems that `renv` doens't help with, and how to uninstall it, if you no longer want to use it.
 
-Underlying the philosophy of `renv` is that any of your existing workflows
-should just work as they did before -- `renv` helps manage library paths (and
-other project-specific state) to help isolate your project's R dependencies,
-and the existing tools you've used for managing R packages (e.g.
-`install.packages()`, `remove.packages()`) should work as they did before.
+This vignette assumes that you're are already living a project-centric lifestyle and are familiar with a version control system, like Git and GitHub: we believe these are table stakes for reproducible data science.
+If you're not already using projects, we recommend [Workflow: Projects](https://r4ds.had.co.nz/workflow-projects.html) from *R for Data Science*; if you're unfamiliar with [Git](https://git-scm.com/) and [GitHub](https://github.com/), we recommend [Happy Git and GitHub for useR](https://happygitwithr.com).
 
+## Libraries and repositories
 
-## Workflow
+Before we get started there are two important pieces of R jargon that you'll need to fully understand: libraries and repositories.
 
-The general workflow when working with `renv` is:
+A **library** is a directory containing installed packages.
+This term is confusing because you write `library(dplyr)`, making it easy to think of dplyr as a library, not a package.
+And that confusing doesn't normally matter because you don't usually think too much about libraries.
+But a project-specific library is a critical part of the way `renv` works, so you'll now need to get the distinction clear in your head.
+You can see your current libraries with `.libPaths()` and see which packages are available in each library with `lapply(.libPaths(), list.files)`.
```

Comment:
Because I want to emphasise that packages are just directories here; I don't think it's too confusing if they see a couple of entries that aren't packages.